src/components/pickers/types.tsx

```diff
@@ -1,9 +1,52 @@
-import { ControllerRenderProps } from 'react-hook-form';
+import { ControllerRenderProps, FieldValues } from 'react-hook-form';
 
 import { ICustomerMaster } from 'models/customerMaster/ICustomerMaster';
 import { IDriver } from 'models/driver/IDriver';
 
+export type TransporterPickerProp = {
+  field: ControllerRenderProps<FieldValues, 'transporter'>;
+  isInvalid: boolean;
+};
 export type PickerProp = {
   field: ControllerRenderProps<ICustomerMaster, 'customer_type'> | ControllerRenderProps<IDriver, 'transporter'>
   isInvalid: boolean;
 };
+
+export type DriverPickerProp = {
+  field: ControllerRenderProps<FieldValues, 'driver'>;
+  isInvalid: boolean;
+  watch?: any;
+  isShipment?: boolean;
+  setValue: (name: string, value: unknown, config?: Object) => void;
+  mode?: string; // possible values: 'Edit', 'Add', ''
```

Comment:
I think is possible to write `mode?: 'Edit' | 'Add' | ''` and like that just those types can pass. BTW "edit"/"add"

Reply:
yes, we can do this later, this is temporary solution.